### ISSUE TITLE
Bug switch présence 

### ISSUE THREAD

**Issue by djrams10** [chrisjshull/homebridge-nest#494]
**Describe the bug**

After restarting Homebridge all works well but after about two days, when I leave my home and that the presence switch turns off, the eco switch trips and turns off immediately, as well as the presence switch.
If I turn on and I come in and I turn on the eco switch, it does not pose a particular problem. This is the presence switch that works not bad.
As soon as this problem arises if I turn on or off the presence switch, it no longer reacts to the iOS Nest application, it remains on absent. Thank you for helping me it hard for a long time, I restart Homebridge and all come back in order. And this problem has been present for several releases. I must be the only one I do not see the problem by norm elsewhere. Thank you. (sorry for my English, I am French).

https://user-images.githubusercontent.com/28505910/150405651-4d3523a8-32a6-49cc-b26b-45e490c66a8e.MOV

**To Reproduce**
1. Activate the presence switch

**Expected behavior**

Include with your bug report this version info:
```sh
node --version 14.15.1
homebridge --version 1.3.9

Logs after problems:

[1/20/2022, 8:13:20 PM] [Nest] getTargetTemperature heat off 21
[1/20/2022, 8:13:23 PM] [Nest] -> Protobuf (Cmd) - {
  "resourceCommands": [
    {
      "traitLabel": "structure_mode",
      "command": {
        "type_url": "type.nestlabs.com/nest.trait.occupancy.StructureModeTrait.StructureModeChangeRequest",
        "value": {
          "structureMode": "STRUCTURE_MODE_AWAY",
          "reason": "STRUCTURE_MODE_REASON_EXPLICIT_INTENT",
          "userId": {
            "resourceId": "USER_015EB5C8DAC6A6B3"
          }
        }
      }
    }
  ],
  "resourceRequest": {
    "resourceId": "STRUCTURE_015EB5C8DAC6A6B4",
    "requestId": "40a7617c-4d97-4eab-a85a-a00b7713ae95"
  }
}
[1/20/2022, 8:13:23 PM] [Nest] Send command request: CkIKGlNUUlVDVFVSRV8wMTVFQjVDOERBQzZBNkI0EiQ0MGE3NjE3Yy00ZDk3LTRlYWItYTg1YS1hMDBiNzcxM2FlOTUShwEKDnN0cnVjdHVyZV9tb2RlEnUKVHR5cGUubmVzdGxhYnMuY29tL25lc3QudHJhaXQub2NjdXBhbmN5LlN0cnVjdHVyZU1vZGVUcmFpdC5TdHJ1Y3R1cmVNb2RlQ2hhbmdlUmVxdWVzdBIdCAIQARoXChVVU0VSXzAxNUVCNUM4REFDNkE2QjM=
[1/20/2022, 8:13:25 PM] [Nest] --> Command response: {
  "resouceCommandResponse": [
    {
      "resourceRequest": {
        "resourceId": "STRUCTURE_015EB5C8DAC6A6B4",
        "requestId": "40a7617c-4d97-4eab-a85a-a00b7713ae95"
      },
      "traitOperations": [
        {
          "traitRequest": {
            "resourceId": "STRUCTURE_015EB5C8DAC6A6B4",
            "traitLabel": "structure_mode",
            "requestId": "40a7617c-4d97-4eab-a85a-a00b7713ae95"
          },
          "progress": "COMPLETE",
          "event": {
            "event": {
              "responseType": "STRUCTURE_MODE_CHANGE_RESPONSE_TYPE_SUCCESS",
              "@type": "type.nestlabs.com/nest.trait.occupancy.StructureModeTrait.StructureModeChangeResponse"
            }
          },
          "command": {
            "traitRequest": {
              "resourceId": "STRUCTURE_015EB5C8DAC6A6B4",
              "traitLabel": "structure_mode",
              "requestId": "40a7617c-4d97-4eab-a85a-a00b7713ae95"
            },
            "command": {
              "structureMode": "STRUCTURE_MODE_AWAY",
              "reason": "STRUCTURE_MODE_REASON_EXPLICIT_INTENT",
              "userId": {
                "resourceId": "USER_015EB5C8DAC6A6B3"
              },
              "@type": "type.nestlabs.com/nest.trait.occupancy.StructureModeTrait.StructureModeChangeRequest"
            },
            "expiryTime": {
              "seconds": "1642706064",
              "nanos": 36000000
            }
          }
        }
      ]
    }
  ],
  "unknown": ""
}
[1/20/2022, 8:13:25 PM] [homebridge-nest] Characteristic 'On': SET handler returned write response value, though the characteristic doesn't support write response. See https://git.io/JtMGR for more info.
[1/20/2022, 8:13:25 PM] [homebridge-nest] Error: 
    at On.Characteristic.characteristicWarning (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Characteristic.ts:2074:105)
    at /usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Characteristic.ts:1699:22
    at /usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/util/once.ts:9:18
    at /homebridge/node_modules/homebridge-nest/index.js:16:22
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[1/20/2022, 8:13:25 PM] [Nest] Home Occupied for Home Occupied is: false
[1/20/2022, 8:13:26 PM] [Nest] Setting Eco Mode for Living Room Thermostat to: eco
[1/20/2022, 8:13:26 PM] [Nest] Setting target heating cooling for Living Room Thermostat to: eco
[1/20/2022, 8:13:26 PM] [Nest] shared.09AA01AC06170HSB hvac_mode eco
[1/20/2022, 8:13:26 PM] [Nest] Committing update device.09AA01AC06170HSB { eco: { mode: 'manual-eco' } }
[1/20/2022, 8:13:26 PM] [Nest] Pushing updates 1 in 0 ms
[1/20/2022, 8:13:26 PM] [Nest] 0 - {
  object_key: 'device.09AA01AC06170HSB',
  op: 'MERGE',
  value: { eco: { mode: 'manual-eco' } }
}
[1/20/2022, 8:13:26 PM] [homebridge-nest] Characteristic 'On': SET handler returned write response value, though the characteristic doesn't support write response. See https://git.io/JtMGR for more info.
[1/20/2022, 8:13:26 PM] [homebridge-nest] Error: 
    at On.Characteristic.characteristicWarning (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Characteristic.ts:2074:105)
    at /usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Characteristic.ts:1699:22
    at /usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/util/once.ts:9:18
    at /homebridge/node_modules/homebridge-nest/index.js:16:22
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[1/20/2022, 8:13:26 PM] [Nest] Eco Mode for Living Room Thermostat is: true
[1/20/2022, 8:13:27 PM] [Nest] device.09AA01AC06170HSB/eco: overriding { mode: 'manual-eco' } -> { mode: 'manual-eco' }
[1/20/2022, 8:13:27 PM] [Nest] API subscribe GET: got updated data
[1/20/2022, 8:13:27 PM] [Nest] getTargetTemperature eco off 21
[1/20/2022, 8:13:27 PM] [Nest] Target temperature for Living Room Thermostat is: 18.5 °C / 65.3 °F
[1/20/2022, 8:13:27 PM] [Nest] Cooling threshold temperature for Living Room Thermostat is: 24.5 °C / 76.10000000000001 °F
[1/20/2022, 8:13:27 PM] [Nest] Heating threshold temperature for Living Room Thermostat is: 18.5 °C / 65.3 °F
[1/20/2022, 8:13:27 PM] [Nest] Home Occupied for Home Occupied is: true
[1/20/2022, 8:13:27 PM] [Nest] API subscribe POST: subscribing
[1/20/2022, 8:13:27 PM] [Nest] Setting Eco Mode for Living Room Thermostat to: heat
[1/20/2022, 8:13:27 PM] [Nest] Setting target heating cooling for Living Room Thermostat to: eco-off
[1/20/2022, 8:13:27 PM] [Nest] shared.09AA01AC06170HSB hvac_mode eco-off
[1/20/2022, 8:13:27 PM] [Nest] Committing update device.09AA01AC06170HSB { eco: { mode: 'schedule' } }
[1/20/2022, 8:13:27 PM] [Nest] Pushing updates 1 in 0 ms
[1/20/2022, 8:13:27 PM] [Nest] 0 - {
  object_key: 'device.09AA01AC06170HSB',
  op: 'MERGE',
  value: { eco: { mode: 'schedule' } }
}
[1/20/2022, 8:13:27 PM] [homebridge-nest] Characteristic 'On': SET handler returned write response value, though the characteristic doesn't support write response. See https://git.io/JtMGR for more info.
[1/20/2022, 8:13:27 PM] [homebridge-nest] Error: 
    at On.Characteristic.characteristicWarning (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Characteristic.ts:2074:105)
    at /usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Characteristic.ts:1699:22
    at /usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/util/once.ts:9:18
    at /homebridge/node_modules/homebridge-nest/index.js:16:22
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[1/20/2022, 8:13:27 PM] [Nest] Eco Mode for Living Room Thermostat is: false
[1/20/2022, 8:13:27 PM] [Nest] getTargetTemperature heat off 21
[1/20/2022, 8:13:27 PM] [Nest] Target temperature for Living Room Thermostat is: 21 °C / 69.8 °F
[1/20/2022, 8:13:27 PM] [Nest] setTargetTemperature heat 21
[1/20/2022, 8:13:27 PM] [Nest] Setting temperature to 21 °C / 69.8 °F
[1/20/2022, 8:13:27 PM] [Nest] Setting target temperature for Living Room Thermostat to: 21
[1/20/2022, 8:13:27 PM] [Nest] shared.09AA01AC06170HSB target_temperature 21
[1/20/2022, 8:13:27 PM] [Nest] Committing update shared.09AA01AC06170HSB { target_temperature: 21 }
[1/20/2022, 8:13:27 PM] [homebridge-nest] Characteristic 'Target Temperature': SET handler returned write response value, though the characteristic doesn't support write response. See https://git.io/JtMGR for more info.
[1/20/2022, 8:13:27 PM] [homebridge-nest] Error: 
    at TargetTemperature.Characteristic.characteristicWarning (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Characteristic.ts:2074:105)
    at /usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Characteristic.ts:1699:22
    at /usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/util/once.ts:9:18
    at /homebridge/node_modules/homebridge-nest/index.js:16:22
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[1/20/2022, 8:13:27 PM] [Nest] Target temperature for Living Room Thermostat is: 21 °C / 69.8 °F
[1/20/2022, 8:13:28 PM] [Nest] device.09AA01AC06170HSB/eco: overriding { mode: 'schedule' } -> { mode: 'manual-eco' }
[1/20/2022, 8:13:28 PM] [Nest] device.09AA01AC06170HSB/eco: overriding { mode: 'manual-eco' } -> { mode: 'schedule' }
[1/20/2022, 8:13:28 PM] [Nest] shared.09AA01AC06170HSB/target_temperature: overriding 21 -> 21
[1/20/2022, 8:13:28 PM] [Nest] API subscribe GET: got updated data
[1/20/2022, 8:13:28 PM] [Nest] getTargetTemperature heat off 21
[1/20/2022, 8:13:28 PM] [Nest] Cooling threshold temperature for Living Room Thermostat is: 24 °C / 75.2 °F
[1/20/2022, 8:13:28 PM] [Nest] Heating threshold temperature for Living Room Thermostat is: 20 °C / 68 °F
[1/20/2022, 8:13:28 PM] [Nest] API subscribe POST: subscribing
[1/20/2022, 8:13:29 PM] [Nest] device.09AA01AC06170HSB/eco: overriding {
  mode: 'manual-eco',
  touched_by: 0,
  mode_update_timestamp: 1642706008
} -> { mode: 'manual-eco' }
[1/20/2022, 8:13:29 PM] [Nest] device.09AA01AC06170HSB/eco: overriding { mode: 'manual-eco' } -> { mode: 'schedule' }
[1/20/2022, 8:13:29 PM] [Nest] shared.09AA01AC06170HSB/target_temperature: overriding 21 -> 21
[1/20/2022, 8:13:29 PM] [Nest] API subscribe GET: got updated data
[1/20/2022, 8:13:29 PM] [Nest] Current humidity for Living Room Thermostat is: 44%
[1/20/2022, 8:13:29 PM] [Nest] getTargetTemperature heat off 21
[1/20/2022, 8:13:29 PM] [Nest] API subscribe POST: subscribing
[1/20/2022, 8:13:29 PM] [Nest] device.09AA01AC06170HSB/eco: overriding {
  mode: 'manual-eco',
  touched_by: 0,
  mode_update_timestamp: 1642706008
} -> { mode: 'manual-eco' }
[1/20/2022, 8:13:29 PM] [Nest] device.09AA01AC06170HSB/eco: overriding { mode: 'manual-eco' } -> { mode: 'schedule' }
[1/20/2022, 8:13:29 PM] [Nest] shared.09AA01AC06170HSB/target_temperature: overriding 21 -> 21
[1/20/2022, 8:13:29 PM] [Nest] API subscribe GET: got updated data
[1/20/2022, 8:13:29 PM] [Nest] getTargetTemperature heat off 21
[1/20/2022, 8:13:29 PM] [Nest] API subscribe POST: subscribing
[1/20/2022, 8:13:29 PM] [Nest] Pushing updates 1 in 4718 ms
[1/20/2022, 8:13:29 PM] [Nest] 0 - {
  object_key: 'shared.09AA01AC06170HSB',
  op: 'MERGE',
  value: { target_temperature: 21 }
}
[1/20/2022, 8:13:30 PM] [Nest] device.09AA01AC06170HSB/eco: overriding {
  mode: 'manual-eco',
  touched_by: 0,
  mode_update_timestamp: 1642706008
} -> { mode: 'manual-eco' }
[1/20/2022, 8:13:30 PM] [Nest] device.09AA01AC06170HSB/eco: overriding { mode: 'manual-eco' } -> { mode: 'schedule' }
[1/20/2022, 8:13:30 PM] [Nest] shared.09AA01AC06170HSB/target_temperature: overriding 21 -> 21
[1/20/2022, 8:13:30 PM] [Nest] API subscribe GET: got updated data
[1/20/2022, 8:13:30 PM] [Nest] getTargetTemperature heat off 21
[1/20/2022, 8:13:30 PM] [Nest] API subscribe POST: subscribing
[1/20/2022, 8:13:35 PM] [Nest] device.09AA01AC06170HSB/eco: overriding {
  mode: 'manual-eco',
  touched_by: 0,
  mode_update_timestamp: 1642706008
} -> { mode: 'schedule' }
[1/20/2022, 8:13:35 PM] [Nest] shared.09AA01AC06170HSB/target_temperature: overriding 21 -> 21
[1/20/2022, 8:13:35 PM] [Nest] API subscribe GET: got updated data
[1/20/2022, 8:13:35 PM] [Nest] getTargetTemperature heat off 21
[1/20/2022, 8:13:35 PM] [Nest] API subscribe POST: subscribing
[1/20/2022, 8:13:35 PM] [Nest] Re-syncing with Nest API state.
[1/20/2022, 8:13:35 PM] [Nest] getTargetTemperature eco off 21
[1/20/2022, 8:13:35 PM] [Nest] Target temperature for Living Room Thermostat is: 18.5 °C / 65.3 °F
[1/20/2022, 8:13:35 PM] [Nest] Cooling threshold temperature for Living Room Thermostat is: 24.5 °C / 76.10000000000001 °F
[1/20/2022, 8:13:35 PM] [Nest] Heating threshold temperature for Living Room Thermostat is: 18.5 °C / 65.3 °F
[1/20/2022, 8:13:35 PM] [Nest] Eco Mode for Living Room Thermostat is: true
[1/20/2022, 8:13:36 PM] [Nest] API subscribe GET: got updated data
[1/20/2022, 8:13:36 PM] [Nest] getTargetTemperature eco off 21
[1/20/2022, 8:13:36 PM] [Nest] API subscribe POST: subscribing
[1/20/2022, 8:13:36 PM] [Nest] API subscribe GET: got updated data
[1/20/2022, 8:13:36 PM] [Nest] getTargetTemperature eco off 21
[1/20/2022, 8:13:36 PM] [Nest] API subscribe POST: subscribing
[1/20/2022, 8:13:41 PM] [Nest] API observe PING: request error ERR_HTTP2_INVALID_SESSION
[1/20/2022, 8:14:41 PM] [Nest] API observe PING: request error ERR_HTTP2_INVALID_SESSION
[1/20/2022, 8:15:36 PM] [Nest] API subscribe POST: subscribing
[1/20/2022, 8:15:38 PM] [Nest] getTargetTemperature eco off 21
[1/20/2022, 8:15:41 PM] [Nest] API observe PING: request error ERR_HTTP2_INVALID_SESSION
[1/20/2022, 8:16:41 PM] [Nest] API observe PING: request error ERR_HTTP2_INVALID_SESSION
[1/20/2022, 8:17:12 PM] [Nest] API subscribe GET: got updated data
[1/20/2022, 8:17:12 PM] [Nest] getTargetTemperature eco off 21
[1/20/2022, 8:17:12 PM] [Nest] API subscribe POST: subscribing
[1/20/2022, 8:17:17 PM] [Nest] getTargetTemperature eco off 21
[1/20/2022, 8:17:17 PM] [Nest] getTargetTemperature eco off 21
[1/20/2022, 8:17:41 PM] [Nest] API observe PING: request error ERR_HTTP2_INVALID_SESSION

**Comment by adriancable** [comment]
@djrams10 - thank you for filing this! And also, thank you so much for including the log and the video which makes it exactly possible for me to see what you are describing. I wish more people filing issues would do that.

While the effect you are observing is not exactly the same, this does seem to be caused by the same issue as #467. I am currently trialling a fix for that, and it should go into the next version of the plug-in update if all is OK. I will keep you updated here.

**Comment by djrams10** [comment]
Thank you, it's completely normal given the work you 

Here is the line of the logs that triggers the problem:
API observe PING: request error ERR HTTP2 INVALID SESSION
After reboot here is the logs without problem:
6:30:22 AM] [Nest] API observes PING: 12.90199 ms, payload 'd083363c
[1/21/2022,
6:31:22 AM]
[Nest] API observes PING: 12.712986 ms, payload
'479bdc2b
[1/21/2022,
6:31:23 AM]
[Nest] API observe POST: complete message received (2)

Thank you, we look forward to hearing from you. See you soon

**Comment by djrams10** [comment]
hello, I contact you to communicate the logs to you when the bug is produced. thanks to you. See you soon.

[1/25/2022, 11:05:41 AM] [Nest] API observe PING: 12.825377 ms, payload '36a68440'
[1/25/2022, 11:06:41 AM] [Nest] API observe PING: 13.14627 ms, payload '5c2b3e5a'
[1/25/2022, 11:07:27 AM] [Nest] API subscribe POST: subscribing
[1/25/2022, 11:07:41 AM] [Nest] API observe PING: 12.987018 ms, payload '316e7847'
[1/25/2022, 11:08:41 AM] [Nest] API observe PING: 13.107207 ms, payload 'd9bad40f'
[1/25/2022, 11:09:27 AM] [Nest] API subscribe POST: subscribing
[1/25/2022, 11:09:41 AM] [Nest] API observe PING: 13.110882 ms, payload '12ef840f'
[1/25/2022, 11:09:42 AM] [Nest] API observe POST: complete message received (2)
[1/25/2022, 11:09:42 AM] [Nest] getTargetTemperature heat off 21
[1/25/2022, 11:09:56 AM] [Nest] API observe POST: stream error Error: Stream closed with error code NGHTTP2_INTERNAL_ERROR
    at new NodeError (node:internal/errors:371:5)
    at ClientHttp2Stream._destroy (node:internal/http2/core:2313:13)
    at _destroy (node:internal/streams/destroy:102:25)
    at ClientHttp2Stream.destroy (node:internal/streams/destroy:64:5)
    at Http2Stream.onStreamClose (node:internal/http2/core:545:12) {
  code: 'ERR_HTTP2_STREAM_ERROR'
}
[1/25/2022, 11:09:56 AM] [Nest] API observe: error Error: Stream closed with error code NGHTTP2_INTERNAL_ERROR
    at new NodeError (node:internal/errors:371:5)
    at ClientHttp2Stream._destroy (node:internal/http2/core:2313:13)
    at _destroy (node:internal/streams/destroy:102:25)
    at ClientHttp2Stream.destroy (node:internal/streams/destroy:64:5)
    at Http2Stream.onStreamClose (node:internal/http2/core:545:12) {
  code: 'ERR_HTTP2_STREAM_ERROR'
}
[1/25/2022, 11:09:56 AM] [Nest] Retrying in 10 seconds.
[1/25/2022, 11:09:59 AM] [Nest] API subscribe GET: got updated data
[1/25/2022, 11:09:59 AM] [Nest] getTargetTemperature heat off 21
[1/25/2022, 11:09:59 AM] [Nest] API subscribe POST: subscribing
[1/25/2022, 11:10:06 AM] [Nest] API observe: setting issue timer.
[1/25/2022, 11:10:06 AM] [Nest] API observe POST: streaming request initiated
[1/25/2022, 11:10:06 AM] [Nest] API observe POST: issuing
[1/25/2022, 11:10:06 AM] [Nest] API observe cancelled as initiating new call.
[1/25/2022, 11:10:06 AM] [Nest] API observe POST: stream ended
[1/25/2022, 11:10:07 AM] [Nest] API observe POST: complete message received (4090)
[1/25/2022, 11:10:08 AM] [Nest] API observe POST: complete message received (12813)
[1/25/2022, 11:10:08 AM] [Nest] Protobuf message object length 1
[1/25/2022, 11:10:08 AM] [Nest] Protobuf updated properties: user_nfc_token_management@USER_015EB5C8DAC6A6B3 (type.nestlabs.com/weave.trait.security.UserNFCTokenManagementTrait), user_nfc_tokens_data@USER_015EB5C8DAC6A6B3 (type.nestlabs.com/weave.trait.security.UserNFCTokensTrait), gaia_merge_notifications@USER_015EB5C8DAC6A6B3 (type.nestlabs.com/nest.trait.olive.GaiaMergeNotificationsTrait), user_security_notification_settings@USER_015EB5C8DAC6A6B3 (type.nestlabs.com/nest.trait.user.UserSecurityNotificationSettingsTrait), user_lock_notification_settings@USER_015EB5C8DAC6A6B3 (type.nestlabs.com/nest.trait.user.UserLockNotificationSettingsTrait), user_info@USER_015EB5C8DAC6A6B3 (type.nestlabs.com/nest.trait.user.UserInfoTrait), application_keys@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/weave.trait.auth.ApplicationKeysTrait), basic_structure_pincode_schedules_settings@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/weave.trait.schedule.BasicUserSchedulesSettingsTrait), user_pincodes@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/weave.trait.security.UserPincodesSettingsTrait), user_nfc_tokens@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/weave.trait.security.UserNFCTokensTrait), timezone@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/weave.trait.time.TimezoneTrait), peer_devices@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/weave.trait.peerdevices.PeerDevicesTrait), guests@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/nest.trait.guest.GuestsTrait), user_access_records@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/nest.trait.user.UserAccessTrait), pairing@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/nest.trait.pairing.PairingTrait), security_settings@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/nest.trait.security.SecuritySettingsTrait), structure_mode@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/nest.trait.occupancy.StructureModeTrait), structure_mode_capabilities@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/nest.trait.occupancy.StructureModeCapabilitiesTrait), distributed_security@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/nest.trait.security.DistributedSecurityTrait), structure_mode_settings@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/nest.trait.occupancy.StructureModeSettingsTrait), custom_located_annotations@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/nest.trait.located.CustomLocatedAnnotationsTrait), related_resources@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/nest.trait.resourcedirectory.RelatedResourcesTrait), security_action_on_unlock_settings@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/nest.trait.security.SecurityActionOnUnlockSettingsTrait), structure_geofencing@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/nest.trait.occupancy.StructureGeofencingTrait), structure_scenes@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/nest.trait.lighting.StructureScenesTrait), lighting_ecosystem_features_settings@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/nest.trait.lighting.LightingEcosystemFeaturesSettingsTrait), security_action_on_nfc_token_global_settings@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/nest.trait.security.SecurityActionOnNFCTokenGlobalSettingsTrait), lighting_action_on_security_alarm_global_settings@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/nest.trait.lighting.LightingActionOnSecurityAlarmGlobalSettingsTrait), lighting_action_on_safety_alarm_global_settings@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/nest.trait.lighting.LightingActionOnSafetyAlarmGlobalSettingsTrait), fan_action_on_smoke_alarm_global_settings@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/nest.trait.lighting.FanActionOnSmokeAlarmGlobalSettingsTrait), security_action_on_nfc_token_settings@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/nest.trait.security.SecurityActionOnNFCTokenSettingsTrait), lighting_action_on_security_alarm_settings@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/nest.trait.lighting.LightingActionOnSecurityAlarmSettingsTrait), lighting_action_on_safety_alarm_settings@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/nest.trait.lighting.LightingActionOnSafetyAlarmSettingsTrait), fan_action_on_smoke_alarm_settings@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/nest.trait.lighting.FanActionOnSmokeAlarmSettingsTrait), motion_alerts_global_settings@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/nest.trait.lighting.MotionAlertsGlobalSettingsTrait), home_info_settings@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/nest.trait.structure.HomeInfoSettingsTrait), enterprise_programs@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/nest.trait.hvac.EnterpriseProgramsEntitlementsTrait), utility_settings@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/nest.trait.hvac.UtilitySettingsTrait), safety_shutoff_settings@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/nest.trait.hvac.SafetyShutoffSettingsTrait), seasonal_savings_action@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/nest.trait.hvac.SeasonalSavingsActionTrait), seasonal_savings_settings@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/nest.trait.hvac.SeasonalSavingsSettingsTrait), demand_response_configuration@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/nest.trait.hvac.DemandResponseConfigurationTrait), located_annotations@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/nest.trait.located.LocatedAnnotationsTrait), structure_info@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/nest.trait.structure.StructureInfoTrait), structure_location@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/nest.trait.structure.StructureLocationTrait), protect_safety_summary@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/nest.trait.product.protect.SafetySummaryTrait), occupancy_history@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/nest.trait.history.OccupancyHistoryTrait), protect_legacy_structure_self_test@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/nest.trait.product.protect.LegacyStructureSelfTestTrait), protect_legacy_self_test_settings@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/nest.trait.product.protect.LegacySelfTestSettingsTrait), safety_structure_settings@STRUCTURE_015EB5C8DAC6A6B4 (type.nestlabs.com/nest.trait.product.protect.SafetyStructureSettingsTrait)
[1/25/2022, 11:10:08 AM] [Nest] Legacy user mapping USER_015EB5C8DAC6A6B3 -> user.5819408
[1/25/2022, 11:10:08 AM] [Nest] Protobuf API: structure 3f9b51a0-1157-11e7-bede-22000bca4566, found 0 device(s)
[1/25/2022, 11:10:08 AM] [Nest] getTargetTemperature heat off 21
[1/25/2022, 11:10:16 AM] [Nest] getTargetTemperature heat off 21
[1/25/2022, 11:10:26 AM] [Nest] API subscribe GET: got updated data
[1/25/2022, 11:10:26 AM] [Nest] getTargetTemperature heat off 21
[1/25/2022, 11:10:26 AM] [Nest] API subscribe POST: subscribing
[1/25/2022, 11:10:40 AM] [Nest] Initiating pre-emptive reauthentication.
[1/25/2022, 11:10:40 AM] [Nest] Authenticating via Google.
[1/25/2022, 11:10:40 AM] [Nest] API observe PING: request error ERR_HTTP2_INVALID_SESSION
[1/25/2022, 11:10:42 AM] [Nest] Authentication successful.
[1/25/2022, 11:11:40 AM] [Nest] API observe PING: request error ERR_HTTP2_INVALID_SESSION
[1/25/2022, 11:11:52 AM] [Nest] API observe POST: session/request timed out
[1/25/2022, 11:12:26 AM] [Nest] API subscribe POST: subscribing
[1/25/2022, 11:12:40 AM] [Nest] API observe PING: request error ERR_HTTP2_INVALID_SESSION
[1/25/2022, 11:13:40 AM] [Nest] API observe PING: request error ERR_HTTP2_INVALID_SESSION

**Comment by adriancable** [comment]
Fixed in 4.6.4, now released.
